recipes-multimedia/gstreamer/gstreamer1.0-plugins-base_1.24.%.bbappend

```diff
@@ -13,5 +13,5 @@ PACKAGECONFIG_GL:use-mainline-bsp = \
 # does not depend on the viv-fb feature. It used to, but that was actually a bug
 # which was fixed in GStreamer 1.22.5. Since then, the direct texture support is
 # detected by Meson by checking for direct texture symbols like "glTexDirectVIV".)
-PACKAGECONFIG_GL:append:mx6-nxp-bsp = " viv-fb "
-PACKAGECONFIG_GL:append:mx7ulp-nxp-bsp = " viv-fb "
+PACKAGECONFIG_GL:imxgpu2d:append:mx6-nxp-bsp = " viv-fb "
+PACKAGECONFIG_GL:imxgpu2d:append:mx7-nxp-bsp = " viv-fb "
```

Comment:
I think this not works, the `append` needs to be the first override in the list. Can you please verify the `PACKAGECONFIG_GL` on any of the referred machines ?

Reply:
It is necessary, these are appending to the assignments on lines 1-4.

Perhaps it could be clearer by dropping the `append` and adding a variable?

```
PACKAGECONFIG_GL:imxgpu2d = " \
    ${@bb.utils.contains('DISTRO_FEATURES', 'opengl x11', 'opengl', '', d)} \
    ${PACKAGECONFIG_GL_VIV} \
"
PACKAGECONFIG_GL:imxgpu3d = " \
    ${@bb.utils.contains('DISTRO_FEATURES', 'opengl', 'gles2 egl', '', d)} \
    ${PACKAGECONFIG_GL_VIV} \
"
PACKAGECONFIG_GL:use-mainline-bsp = \
    "${@bb.utils.contains('DISTRO_FEATURES', 'opengl', 'gles2 egl gbm', '', d)}"

# The i.MX8 uses KMS instead of the Vivante specific framebuffer API.
# The i.MX7 does not have a GPU, except for ULP.
# This leaves the i.MX6 - with the vendor BSP - as the remaining use case for viv-fb.
#
# (Note that viv-fb is about the _windowing system_. Vivante direct texture support
# does not depend on the viv-fb feature. It used to, but that was actually a bug
# which was fixed in GStreamer 1.22.5. Since then, the direct texture support is
# detected by Meson by checking for direct texture symbols like "glTexDirectVIV".)
PACKAGECONFIG_GL_VIV             = ""
PACKAGECONFIG_GL_VIV:mx6-nxp-bsp = " viv-fb "
PACKAGECONFIG_GL_VIV:mx7-nxp-bsp = " viv-fb "
```

Reply:
My concern is just about the `append` positions, I think it should be
```patch
-PACKAGECONFIG_GL:imxgpu2d:append:mx6-nxp-bsp = " viv-fb "
-PACKAGECONFIG_GL:imxgpu2d:append:mx7-nxp-bsp = " viv-fb "
+PACKAGECONFIG_GL:append:imxgpu2d:mx6-nxp-bsp = " viv-fb "
+PACKAGECONFIG_GL:append:imxgpu2d:mx7-nxp-bsp = " viv-fb "
```
but have tested locally and it works in the two cases. Sorry for the noise